sdks/python/container/build.gradle

```diff
@@ -75,6 +75,12 @@ task pushAll {
   dependsOn ':sdks:python:container:py38:dockerPush'
 }
 
+task generatePythonRequirementsAll {
+  dependsOn ':sdks:python:container:py36:generatePythonRequirements'
```

Comment:
just curious if the requirements are now different between python versions.  Why is that now the case?

Reply:
they were already different:

1) Certain packages make sense to install only on some versions of python, example: https://github.com/apache/beam/blob/7d54488eca45743e379f1873eda7f6d390f97b50/sdks/python/setup.py#L132

2) some packages don't support some python versions, for example TF does not make new releases on Py 3.6.